yarn.lock

```diff
@@ -11983,27 +12324,27 @@ fsevents@^1.2.7:
   languageName: node
   linkType: hard
 
-"moment@npm:>= 2.6.0, moment@npm:>= 2.9.0, moment@npm:^2.10, moment@npm:^2.11.2, moment@npm:^2.19.1, moment@npm:^2.27.0":
+"moment@npm:>= 2.6.0, moment@npm:>= 2.9.0, moment@npm:^2.10, moment@npm:^2.11.2, moment@npm:^2.19.1":
   version: 2.29.1
   resolution: "moment@npm:2.29.1"
   checksum: 1e14d5f422a2687996be11dd2d50c8de3bd577c4a4ca79ba5d02c397242a933e5b941655de6c8cb90ac18f01cc4127e55b4a12ae3c527a6c0a274e455979345e
   languageName: node
   linkType: hard
 
+"moment@npm:^2.27.0, moment@npm:~2.29.4":
+  version: 2.29.4
```

Comment:
Seems we can bump moment to 2.29.4 to merge this with the above (separate PR of course)

Reply:
100%

Reply:
Linking PR here for continuity https://github.com/ManageIQ/manageiq-ui-classic/pull/8577